oauthproxy.go

```diff
@@ -858,9 +858,17 @@ func (p *OAuthProxy) Proxy(rw http.ResponseWriter, req *http.Request) {
 	session, err := p.getAuthenticatedSession(rw, req)
 	switch err {
 	case nil:
+		//<!--COCO Begin-->
+		// COCO: check session token present in cookie or not
+		cookie, err := req.Cookie("sessionToken")
+		if err != nil {
+			fmt.Printf("can't find sessionToken redireacting to logout page")
+			p.SignOut(rw, req)
```

Comment:
- There is no return after the p.SignOut?

Reply:
yes its there .. but after logout, it's falling into the "/" redirect so I removed it.